connectors/v2/pakartot.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#player';
+
+Connector.artistSelector = 'div.jp-player-artist > a';
+
+Connector.trackSelector = 'div.jp-player-title > a';
+
+Connector.playButtonSelector = '.jp-play';
+
+Connector.getArtist = function() {
+	var text = $(this.artistSelector).text().trim();
```

Comment:
I asked @david-sabata about adding `trim` to default `Connector.getArtist` implementation. He agreed.
I think we'll have no regressions, if we add trimming.